lib/src/widgets/video.dart

```diff
@@ -115,7 +115,8 @@ class Video extends StatefulWidget {
   final bool showTimeLeft;
 
   Video({
-    required this.playerId,
+    @Deprecated('playerId is deprecated. Use player instead.') int? playerId,
```

Comment:
I would've gone with passing `Player` directly in the first place (instead of `playerId`), but when I initially started working on this, there was no sync constructor for creating a `Player`.

Reply:
It's fine, this doesn't have the highest priority. Basically I don't see a need for specifying the player ID upon constructing the player. It should just use the player's memory address (i.e. cast the player's address to an integer ;))